src/main/java/com/knighttodo/knighttodo/rest/DayTodoResource.java

```diff
@@ -0,0 +1,105 @@
+package com.knighttodo.knighttodo.rest;
+
+import static com.knighttodo.knighttodo.Constants.API_BASE_DAYS;
+import static com.knighttodo.knighttodo.Constants.API_BASE_TODOS;
+import static com.knighttodo.knighttodo.Constants.BASE_READY;
+
+import com.knighttodo.knighttodo.domain.DayTodoVO;
+import com.knighttodo.knighttodo.rest.mapper.DayTodoRestMapper;
+import com.knighttodo.knighttodo.rest.request.DayTodoRequestDto;
+import com.knighttodo.knighttodo.rest.response.DayTodoReadyResponseDto;
+import com.knighttodo.knighttodo.rest.response.DayTodoResponseDto;
+import com.knighttodo.knighttodo.service.DayTodoService;
+
+import io.swagger.annotations.ApiOperation;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import javax.validation.Valid;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@Validated
+@RequiredArgsConstructor
+@RestController
+@RequestMapping(API_BASE_DAYS + "/{dayId}" + API_BASE_TODOS)
+@Slf4j
+public class DayTodoResource {
+
+    private final DayTodoService dayTodoService;
+    private final DayTodoRestMapper dayTodoRestMapper;
+
+    @PostMapping
+    @ApiOperation(value = "Add new Day Todo")
+    public ResponseEntity<DayTodoResponseDto> addDayTodo(@PathVariable String dayId,
```

Comment:
1) you use RestController annotation
so ResponseEntity is optional there

RestController =  @Controller + @ResponseBody
for status you can use   @ResponseStatus(HttpStatus.CREATED)

2) it's good style and approach to use 
 - GlobalExceptionIntercepter
 - provide try catch in controller level for specific exception likes 



   ` @PostMapping
    @ResponseStatus(HttpStatus.CREATED)
    @ApiOperation(value = "Create new ToDo", response = DayTodoResponseDto.class)
    @ApiResponses(value = {
            @ApiResponse(code = 201, message = "Created"),
            @ApiResponse(code = 400, message = "Invalid payload or operation"),
            @ApiResponse(code = 403, message = "Operation forbidden"),
            @ApiResponse(code = 404, message = "Resource not found"),
            @ApiResponse(code = 500, message = "Unexpected error")
    })
    public DayTodoResponseDto create(@Valid @RequestBody DayTodoCreateCommand command) {
        try {
          .......
            return response;

        } catch (PermissionsException | ResourceNotFoundException | ResourceCreationException ex) {
            logger.error(ex.getMessage(), ex);
            throw ex;
        } catch (RuntimeException ex) {
            logger.error("DayTodo isn't created", ex);
            throw new CreateTodoDayException(ex);
        }
    }`

Reply:
Thanks. I've created separate tasks for both of suggestions.